src/node_util.cc

```diff
@@ -28,7 +28,8 @@ using v8::Value;
   V(isRegExp, IsRegExp)                                                       \
   V(isSet, IsSet)                                                             \
   V(isSetIterator, IsSetIterator)                                             \
-  V(isTypedArray, IsTypedArray)
+  V(isTypedArray, IsTypedArray)                                               \
+  V(isSharedArrayBuffer, IsSharedArrayBuffer)
```

Comment:
Note: If you rebase against master, this one should already be in the list due to #8510. Feel free to sort that in alphabetically, though. :)